app/pulp/app/tasks/__init__.py

```diff
@@ -0,0 +1,2 @@
+from pulp.app.tasks import repository  # noqa
```

Comment:
This could go multiple ways, and I'm not tied to this one. Here's how I settled on this one to try: If we import every single task, each of which is just a function, into the same namespace, then the function names will have to contain additional namespace to avoid collision. For example, ``delete_repository``, ``delete_importer``, ``delete_*``.

Would we do ``from repository import delete_repository``? That already looks slightly silly.

That would mean having a function at ``pulp.app.tasks.repository.delete_repository``. Why not just name it ``pulp.app.tasks.repository.delete`` and import the whole ``repository`` module?

That said, it's just semantics and naming, with minor changes. I'm not sure it matters a lot in any case.

Also, while typing this I've realized that the tasks in this PR should be moved to ``importer`` and ``publisher`` modules.

What do you think?